Settings Page

Snorvisable opened the pull request:
- Added the settings page
- Added LocalStorage class for communication with local storage
- Moved version to settings page
- Moved logout action to settings page

LaurensvanVliet commented:
picture?

Snorvisable commented:
![wp_ss_20160803_0001](https://cloud.githubusercontent.com/assets/8373502/17363393/6c79b4c2-597a-11e6-8ef6-2c0e1e6aca0a.png)

LaurensvanVliet commented:
dude looks fantastic!!

ST-Apps commented:
While the idea it's great and needed, we don't need the `LocalStorage` class. 
Please update the code to use `Utils\SettingsHelper` since all the storage logic is already there (it's part of Template10). 

Waiting for updates on this :) (btw margins are too "big", but we'll fix visual style later).

Pankaj165 commented:
@Snorvisable great job dude. Looks simple and great.

br1luca commented:
Grat job! its also close what i propose!

Snorvisable commented:
Thanks for the feedback @ST-Apps, will update those things. I didn't notice the SettingsHelper, sorry for that.

Snorvisable commented:
@ST-Apps, I updated the local storage communication part, updated the margins and close button.

![wp_ss_20160803_0003](https://cloud.githubusercontent.com/assets/8373502/17364907/50babdb8-5983-11e6-8908-a1f88a29e3ee.png)

Blumster commented:
Looks promising, but the mixed use of tabs and spaces in the code is disgusting. There should be coding guidelines for the project (maybe just put a link to an existing one in the readme), because it will get messy soon. Also: Tab2Space!

Migsi commented:
@Blumster just auto-beautify the whole code ;)

Could you add a setting to toggle between bright and dark map theme too? Since the last commits seem to have toggled it to the bright one, it would be nice to have a choice :)

DeMoMan159 commented:
Amazing work !

profdrmops commented:
This looks very nice

DarkAngelFR commented:
Looks nice
About the map theme : There is an auto switch from 7 to 19 to light mode and dark at "night". I think a "map setting page" should allow us to use or not this mode, also to activate options or not (like trafic view, road vs satellite, etc...). I can open an issue on this and take it in charge? (after your merge so I can add "map option" in this part of the app)

Migsi commented:
@DarkAngelFR Ok, that makes somehow sense :) But wouldn't it be better to have strong contrast that makes it easy to read during the day and a bright one during the night? ^^"

Snorvisable commented:
Merged to remove conflicts.

Snorvisable commented:
Thanks

ST-Apps commented:
Thank you for your help :)
